packages/manager/modules/backup-agent/src/pages/services/dashboard/agent/download/__tests__/AgentDownload.test.tsx

```diff
@@ -0,0 +1,168 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { BACKUP_AGENT_NAMESPACES } from '@/lib';
+import { mockAgentDownloadLinks } from '@/mocks/agents/agentDownloadLinks';
+import { TENANTS_MOCKS } from '@/mocks/tenant/tenants.mock';
+import DownloadAgentPage from '@/pages/services/dashboard/agent/download/AgentDownload.page';
+
+const { useBackupVSPCTenantAgentDownloadLinkMock } = vi.hoisted(() => ({
+  useBackupVSPCTenantAgentDownloadLinkMock: vi.fn(),
+}));
+
+// --- Mock translation ---
+vi.mock('@ovhcloud/ods-components/react', async () => {
+  const actual = await vi.importActual('@ovhcloud/ods-components/react');
+
+  return {
+    ...actual,
+    OdsText: vi
+      .fn()
+      .mockImplementation(({ children }: { children: React.ReactNode }) => <p>{children}</p>),
+    OdsButton: vi
+      .fn()
+      .mockImplementation(
+        ({
+          label,
+          isLoading,
+          isDisabled,
+          ...props
+        }: {
+          children: React.ReactNode;
+          label: string;
+          isLoading: boolean;
+          isDisabled: boolean;
+        }) => (
+          <button data-is-loading={isLoading} data-is-disabled={isDisabled} {...props}>
+            {label}
+          </button>
+        ),
+      ),
```

Comment:
This is more a general remark and not something to fix here, but don't you feel like you repeat those mocks a lot ?
There is ways to centralise this, it would be less code to write everywhere, but above all less code to maintain